.buildkite/agents.json

```diff
@@ -0,0 +1,79 @@
+{
+  "gcp": {
+    "project": "elastic-kibana-ci",
+    "zones": ["us-central1-a", "us-central1-b", "us-central1-c", "us-central1-f"],
+    "serviceAccount": "elastic-buildkite-agent@elastic-kibana-ci.iam.gserviceaccount.com",
+    "imageFamily": "kb-ubuntu",
+    "subnetwork": "buildkite",
+    "disableExternalIp": true,
```

Comment:
Will we be able to access the machines to debug?

Reply:
Yep. I set this up: https://cloud.google.com/iap/docs/using-tcp-forwarding

It just means that you have to use a `gcloud` command to initiate the SSH connection. It also uses your Elastic Google account to do the auth, so we also don't have to mess with populating SSH keys on the boxes or similar.